flink-libraries/flink-table/src/main/scala/org/apache/flink/table/runtime/aggregate/GeneratedAggregations.scala

```diff
@@ -31,35 +32,35 @@ abstract class GeneratedAggregations extends Function {
     *
     * @param accumulators the accumulators (saved in a row) which contains the current
     *                     aggregated results
-    * @param output       output results collected in a row
+    * @param output       output results collected in a command row
     */
-  def setAggregationResults(accumulators: Row, output: Row)
+  def setAggregationResults(accumulators: Row, output: CRow)
```

Comment:
I would change all `CRow` types back to `Row`. Let's leave the code generation as it is and extract the rows in the wrapper classes.